### PR TITLE
Add update notes

### PR DESCRIPTION
**Pull request by ark2002**
Functionality added
- Add New Note
- Update Note

Preview:

https://deploy-preview-5--get-it-noted.netlify.app/

recording

### **Edit : New Changes** 

- A separate pinned list component 

image

### CODE REVIEW

**Review by yogesh- — APPROVED** [review]
Functionality is as in google keep.
- good folder structure
- code is clean and modular

File: src/frontend/components/BasicNoteCard/BasicNoteCard.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    left: 3rem;
+    background-color: var(--white-color);
+    border: 1px solid var(--secondary-color);
+}
+
+.label__list>div {
+    font-size: 1.6rem;
+    width: 7rem;
+    padding: 5px;
+    /* border: 1px solid var(--black-color); */
```
Comment (krgarima) [inline src/frontend/components/BasicNoteCard/BasicNoteCard.css]: If you don't need this, remove it.

File: src/frontend/components/EditableNote/EditableNote.css [hunk 2]
```diff
@@ -7,34 +7,81 @@
...
+    left: 3rem;
+    background-color: var(--white-color);
+    border: 1px solid var(--secondary-color);
+}
+
+.label__list>div {
+    font-size: 1.6rem;
+    width: 7rem;
+    padding: 5px;
+    /* border: 1px solid var(--black-color); */
```
Comment (krgarima) [inline src/frontend/components/EditableNote/EditableNote.css]: same here

File: src/frontend/components/BasicNoteCard/BasicNoteCard.css [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    background-color: var(--secondary-color);
+}
+
+.color__list {
+    position: absolute;
+    top: 4rem;
+    left: 2rem;
+    padding: 1rem;
+    background-color: var(--secondary-color);
+    gap: 1rem
```
Comment (krgarima) [inline src/frontend/components/BasicNoteCard/BasicNoteCard.css:116]: It is okay to miss colon in js file but it can cause problem in a css file

**Review by krgarima — APPROVED** [review]
aaasd
Give some min-height (approx 55vh) to `notespage__section`. It doesn't look good here.
er
You can hide the label once the user has selected an option. Same goes for color palette.
UI is neat and clean. Code is also good.

File: src/frontend/components/BasicNoteCard/BasicNoteCard.css (hunk 3 above)
Comment (ark2002) [reply src/frontend/components/BasicNoteCard/BasicNoteCard.css:116]: Yup ,Ill look into it asap

File: src/frontend/components/EditableNote/EditableNote.css (hunk 2 above)
Comment (ark2002) [reply src/frontend/components/EditableNote/EditableNote.css]: Will remove it missed my eye

**Comment by ark2002** [comment]
> aaasd Give some min-height (approx 55vh) to `notespage__section`. It doesn't look good here. er You can hide the label once the user has selected an option. Same goes for color palette. UI is neat and clean. Code is also good.

Yup agreed would do the recommendated changes

**Pull request merged**